src/Config/ObjectModel/EntityAction.cs

```diff
@@ -5,6 +5,18 @@ namespace Azure.DataApiBuilder.Config.ObjectModel;
 
 public record EntityAction(EntityActionOperation Action, EntityActionFields? Fields, EntityActionPolicy? Policy)
 {
-    public static readonly HashSet<EntityActionOperation> ValidPermissionOperations = new() { EntityActionOperation.Create, EntityActionOperation.Read, EntityActionOperation.Update, EntityActionOperation.Delete };
+    public static readonly HashSet<EntityActionOperation> ValidPermissionOperationsForCosmos = new() {
```

Comment:
it doesn't make sense to introduce a special action for CosmosDB if the same behavior can be achieved through other actions.

Reply:
Clarified on scrum, we agreed to, not to have separate `patch` permission. if `update` operation is there, then it should allow `patch` operation.